osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -350,6 +348,8 @@ public void ScrollToSelected(bool animated = true)
 
         private BeatmapGroup createGroup(BeatmapSetInfo beatmapSet)
         {
+            beatmapSet = manager.Refresh(beatmapSet);

```

Comment:
this should very likely not be here...

Reply:
That is true but so far I haven't found a way to get an up-to-date beatmap otherwise. I'll try again, I'll look into not scheduling the restore event and see what happens.

Reply:
let's figure this out and fix it before merging this.